app/src/main/res/values/strings.xml

```diff
@@ -685,17 +693,21 @@
 
     <!-- Tip displayed on home view explaining how to request the desktop version of a website.
      Argument 1 will be replaced with a line break in the application. -->
-    <string name="tip_request_desktop"><![CDATA[Rather see the full desktop site?%1$s Menu > Request desktop site]]></string>
+    <string name="tip_request_desktop2">Want to see the full version of a site? Switch on Desktop Site from the %1$s menu></string>
```

Comment:
cc: @Delphine What is the placeholder for? Can you also please check the other strings? Thank you!

Reply:
I've checked other strings and they look good, thanks for asking!

But concerning your question about the placeholder... I think a dev or UX would be able to reply that for sure.
I really can't remember (and it's good you call it out, because localizers are going to have a hard time understanding it if they don't get context in the comment).

Reply:
I think this a bug, the placeholder was probably supposed to be `app_name`, so it would read: "Rather see the full desktop site? Firefox Focus Menu > Request desktop site"